testsuite/clash-testsuite.cabal

```diff
@@ -1,62 +1,88 @@
+license:             BSD-2-Clause
 license-file:        LICENSE
 author:              Christiaan Baaij
 maintainer:          Christiaan Baaij <christiaan.baaij@gmail.com>
 copyright:           Copyright © 2015 University of Twente
 category:            Testing
 build-type:          Simple
 -- extra-source-files:
-cabal-version:       >=1.10
 
 flag cosim
    description:
      Run the co-simulation tests
    default: False
    manual: True
 
-executable clash-testsuite
-  main-is:             Main.hs
-  ghc-options:         -Wall -threaded -with-rtsopts=-N
-  other-modules:       Test.Tasty.Program
-                       Test.Tasty.Clash
-  default-extensions:  DeriveDataTypeable
-                       OverloadedStrings
-                       ViewPatterns
-  build-depends:       base               >=4.10     && <5,
-                       data-default       >=0.7      && <0.8,
-                       deepseq            >=1.4      && <1.5,
-                       directory          >=1.2      && <1.4,
-                       filepath           >=1.4      && <1.5,
-                       Glob               >=0.9      && <1.0,
-                       process            >=1.2      && <1.7,
-                       tasty              >=1.2      && <1.3,
-                       tasty-hunit,
-                       neat-interpolation >=0.3      && <0.4,
-                       concurrent-extra   >=0.7      && <0.8,
-                       temporary,
-                       text,
-                       generic-lens,
-                       interpolate,
-
-                       -- Strictly we don't need these dependencies for the
-                       -- testsuite to compile, but we do when running it.
-                       -- Leaving it out will cause the testsuite to compile
-                       -- it anyway so we're better off doing it beforehand.
-                       clash-ghc,
-                       clash-lib,
-                       clash-prelude
+common basic-config
+  default-language: Haskell2010
+  ghc-options: -Wall
+
+  default-extensions:
+    DeriveDataTypeable
+    OverloadedStrings
+    ViewPatterns
+
+  build-depends:
+    base                >=4.10 && <5,
+    aeson,
+    data-default        >=0.7  && <0.8,
+    directory           >=1.2  && <1.4,
+    filepath            >=1.4  && <1.5,
+    ghc,
+    Glob                >=0.9  && <1.0,
+    neat-interpolation  >=0.3  && <0.4,
+    process             >=1.2  && <1.7,
+    tasty               >=1.2  && <1.3,
+    tasty-hunit,
+    temporary,
+    text,
+
+    -- Strictly we don't need these dependencies for the
+    -- testsuite to compile, but we do when running it.
+    -- Leaving it out will cause the testsuite to compile
+    -- it anyway so we're better off doing it beforehand.
+    clash-ghc,
+    clash-lib,
+    clash-prelude
+
   if flag(cosim)
     build-depends:     clash-cosim
 
-  -- hs-source-dirs:
-  default-language:    Haskell2010
```

Comment:
Why is this removed?

Reply:
It isn't (but I admit the diff could be clearer). It's at the top of the `common` stanza

Reply:
Woops, thanks.